stringfunc.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import "strings"
+
+// StringFunc - string manipulation function wrappers
+type StringFunc struct {}
+
+func (t StringFunc) Replace(s, old, new string) string {
```

Comment:
Can you move the `s` arg to the end of the signature? Like this:

```go
func (t StringFunc) replaceAll(old, new, s string) string {
```

Then it can be used to also support [pipelines](https://golang.org/pkg/text/template/#hdr-Pipelines):

```go
Like this:
{{ replaceAll "a" "o" "banana" }}
or this:
{{ "banana" | replaceAll "a" "o" }}
```

A number of other gomplate built-in funcs support this usage, so it'd fit in nicely.